conanfile.py

```diff
@@ -329,6 +329,8 @@ def requirements(self):
             self.requires("libzip/[>=1.7.3 <2.0.0]")
         if self.options.build_framework:
             self.requires("util-linux-libuuid/[>=2.39 <3.0.0]")
+            if self.settings.os == "Macos":
+                self.requires("gettext/0.21") #needed on MacOS 15 by libuuid
```

Comment:
This seems like an issue of libuuid conan package. We never need to require anything not used directly unless addressing upstream version conflicts.